index.html

```diff
@@ -23,5 +30,127 @@ <h1 class="headline">Ismail<br />Courr</h1>
       <button class="headline_btn" type="button">Start collaboration</button>
       <img src="/img/image_geometry_2.svg" alt="seperator" />
     </section>
+
+    <!-- Projects section -->
+
+    <section id="projects_section">
+      <h2 class="projects_headline">Projects</h2>
+      <article class="project">
+        <img class="project_img" src="/img/project1.png" alt="project cover" />
+        <div class="project_body">
+          <h3 class="project_name">Project name goes here</h3>
+          <ul class="project_languages">
+            <li class="project_language">HTML/CSS</li>
+            <li class="project_language">Ruby on Rails</li>
+            <li class="project_language">JavaScript</li>
+          </ul>
+          <button class="project_btn">
+            See this Project <i class="fa fa-arrow-right"></i>
+          </button>
```

Comment:
- [ ] For each project, the button (e.g., "See this project") is a `button` tag that you used correctly. Can you please also follow the button styling: Active, Hover, and Pressed (see Figma design below). This makes your User Experience better.

<img width="173" alt="image" src="https://user-images.githubusercontent.com/77190203/148426041-0bc2db7f-b928-4b31-8233-e61d9a119c32.png">